src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -585,5 +605,24 @@ public void initSendable(SendableBuilder builder) {
         "Odometry Yaw (deg)", () -> getPose().getRotation().getDegrees(), null);
     builder.addDoubleProperty(
         "Front Left Abs Encoder (rad)", frontLeft::getEncoderAbsPositionRad, null);
+
+    builder.addDoubleProperty(
+        "Rear Right Abs Encoder (rad)", rearRight::getEncoderAbsPositionRad, null);
+    builder.addDoubleProperty(
+        "Front Left Module Pos (rad)", () -> frontLeft.getPosition().angle.getRadians(), null);
```

Comment:
What is the difference between the rearRight::getEncoderAbsPositionRad compared to the frontLeft.getPosition().angle.getRadians()?

Is the getPosition.angel.getRadians() more accurate or something? If so, why do we switch ways for some but not all of the sweveModules?

Reply:
Look at what each of those functions does. `getEncoderAbsPositionRad` obviously gets the _absolute_ position. `getPosition` on the other hand returns a `SwerveModuleState` comprised of the driving encoder (which is relative, there is no absolute) and the _relative_ turning encoder. This is done on lines 193-199 of `SwerveModule.java`:

```java
public SwerveModulePosition getPosition() {
    // Apply chassis angular offset to the encoder position to get the position
    // relative to the chassis.
    return new SwerveModulePosition(
        drivingEncoder.getPosition(),
        new Rotation2d(turningEncoder.getPosition() - chassisAngularOffsetRadians));
  }
```